gdnative-core/src/core_types/vector2.rs

```diff
@@ -13,6 +13,27 @@ pub struct Vector2 {
 ///
 /// See the official [`Godot documentation`](https://docs.godotengine.org/en/3.1/classes/class_vector2.html).
 impl Vector2 {
+    /// The zero vector.
+    pub const ZERO: Vector2 = Vector2::new(0.0, 0.0);
+
+    /// A vector with all components set to 1. Typically used for scaling.
+    pub const ONE: Vector2 = Vector2::new(1.0, 1.0);
+
+    /// A vector with all components set to infinity.
```

Comment:
Not sure if the distinction is important to make, but I've heard positive infinity be called `+Infinity` for clarity elsewhere.

Reply:
I can clarify that.

Maybe a general question, what's the use case of this `INF` constant? Some "default" intervals like `[-Vector2.INF, Vector2.INF]` which are interpreted component-wise?

And secondly, we do want to map every single constant that exists in Godot for completeness, independently of how useful it is, right? (also because it may not be easy to measure utility...)

Reply:
> Use case of `INF`... we do want to map every single constant that exists in Godot for completeness, independently of how useful it is, right? (also because it may not be easy to measure utility...)

I don't know, and yes I think you nailed the problem here. I'd err on the side of accepting constants with dubious usefulness because of this. There isn't a downside imo.